Source/AssetRipper.Processing/AnimatorControllers/AnimatorStateContext.cs

```diff
@@ -0,0 +1,190 @@
+using AssetRipper.Assets.Collections;
+using AssetRipper.Assets.Generics;
+using AssetRipper.SourceGenerated.Classes.ClassID_1102;
+using AssetRipper.SourceGenerated.Classes.ClassID_91;
+using AssetRipper.SourceGenerated.Extensions;
+using AssetRipper.SourceGenerated.Subclasses.StateConstant;
+using AssetRipper.SourceGenerated.Subclasses.StateMachineConstant;
+
+namespace AssetRipper.Processing.AnimatorControllers;
+
+public sealed class AnimatorStateContext
```

Comment:
`internal` please. I like to keep the projects encapsulated unless necessary for unit testing.